### PR TITLE
PHP Barcelona Conference 2015

### PR DESCRIPTION
**Pull request by theUniC**
Hi,

I would like to add the upcoming *PHP Barcelona Conference 2015*, in the **php.net** site.

Thanks!
Christian.

### CODE REVIEW

File: archive/entries/2015-06-01-1.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>PHP Barcelona Conference 2015</title>
+  <id>http://php.net/archive/2015.php#id2015-06-01-1</id>
+  <published>2015-06-01T23:45:28-04:00</published>
+  <updated>2015-06-01T23:45:28-04:00</updated>
+  <default:finalTeaserDate xmlns="http://php.net/ns/news">2015-10-30</default:finalTeaserDate>
+  <category term="conferences" label="Conference announcement"/>
+  <category term="cfp" label="Call for Papers"/>
+  <link href="http://php.net/conferences/index.php#id2015-06-01-1" rel="alternate" type="text/html"/>
+  <default:newsImage xmlns="http://php.net/ns/news" link="http://2015.phpconference.es/" title="PHP Barcelona Conference 2015">phpbarcelonaconference2015.png</default:newsImage>
```
Comment (salathe) [inline archive/entries/2015-06-01-1.xml]: The image filename doesn't match the filename that was committed ("2015" versus "15")

**Comment by salathe** [comment]
Can you resize the image to something in the 300-400 pixel wide range? It's far too big at the moment.

**Comment by theUniC** [comment]
OK! Will do it!

**Comment by theUniC** [comment]
Done! :)

**Comment by php-pulls** [comment]
**Comment on behalf of salathe at php.net:**

Squashed to single commit and merged as 46b09e7

**Pull request closed without merging**